examples/mint-repl/mint-repl.rs

```diff
@@ -0,0 +1,1110 @@
+    let mint_sig = mintinfo
+        .secret_key_set
+        .public_keys()
+        .combine_signatures(mint_sig_shares_ref)
+        .map_err(|e| anyhow!(e))?;
+
+    // Obtain a copy of the tx and sigs from the first MintNode results.
+    let (transaction, transaction_sigs) = results
+        .get(0)
+        .ok_or_else(|| anyhow!("Signature not found"))?;
+
+    // Form the final output DBCs, with Mint's Signature for each.
+    let mut output_dbcs: Vec<Dbc> = reissue_request
+        .transaction
+        .outputs
+        .iter()
+        .map(|content| Dbc {
+            content: content.clone(),
+            transaction: transaction.clone(),
+            transaction_sigs: transaction_sigs
+                .iter()
+                .map(|(input, _)| {
+                    (
+                        *input,
+                        (mintinfo.genesis.owner.public_key(), mint_sig.clone()),
+                    )
+                })
+                .collect(),
+        })
+        .collect();
+
+    // sort outputs by output_number
+    output_dbcs.sort_by_key(|d| d.content.output_number);
+
+    // for each output, construct DbcOwned and display
+    for dbc in output_dbcs.iter() {
+        let pubkeyset = outputs_pks
+            .get(&dbc.content.hash())
+            .ok_or_else(|| anyhow!("PubKeySet not found"))?;
+        let dbc_owned = DbcOwned {
+            inner: dbc.clone(),
+            owner: pubkeyset.clone(),
+        };
+
+        println!("\n-- Begin DBC --");
+        print_dbc_human(&dbc_owned, false)?;
+        println!("-- End DBC --\n");
+    }
+
+    Ok(())
+}
+
+/// Makes a new random SecretKeySet
+fn mk_secret_key_set(threshold: usize) -> Result<(Poly, SecretKeySet)> {
+    let mut rng = rand::thread_rng();
+    let poly = Poly::try_random(threshold, &mut rng).map_err(|e| anyhow!(e))?;
+    Ok((poly.clone(), SecretKeySet::from(poly)))
+}
+
+/// Serialize a SecretKeyShare as big endian bytes
+fn sks_to_bytes(sk: &SecretKeyShare) -> Result<Vec<u8>> {
+    bincode::serialize(&SerdeSecret(&sk))
+        .map(bincode_bytes_to_big_endian_bytes)
+        .map_err(|e| anyhow!(e))
+}
+
+/// Serialize anything serializable as big endian bytes
+fn to_be_bytes<T: Serialize>(sk: &T) -> Result<Vec<u8>> {
+    bincode::serialize(&sk)
+        .map(bincode_bytes_to_big_endian_bytes)
+        .map_err(|e| anyhow!(e))
+}
+
+/// Serialize anything serializable as big endian bytes, hex encoded.
+fn to_be_hex<T: Serialize>(sk: &T) -> Result<String> {
+    Ok(encode(to_be_bytes(sk)?))
+}
+
+/// Deserialize anything deserializable from big endian bytes
+fn from_be_bytes<T: for<'de> Deserialize<'de>>(b: &[u8]) -> Result<T> {
+    let bb = big_endian_bytes_to_bincode_bytes(b.to_vec());
+    bincode::deserialize(&bb).map_err(|e| anyhow!(e))
+}
+
+/// Deserialize anything deserializable from big endian bytes, hex encoded.
+fn from_be_hex<T: for<'de> Deserialize<'de>>(s: &str) -> Result<T> {
+    from_be_bytes(&decode(s)?)
+}
```

Comment:
These could be useful for sn_client once DBC's are exposed there, need to remember they are here.

Reply:
yeah very useful since threshold_crypto api avoids exposing raw bytes.  But really any non-trivial app will require them.